db/db_test.go

```diff
@@ -63,4 +63,79 @@ func TestDB(t *testing.T) {
 			So(res, ShouldResemble, orig)
 		})
 	})
```

Comment:
Remove tests for read/writeGob - they are implementation details, and are now covered by other tests.